Include/internal/pycore_freelist.h

```diff
@@ -67,11 +69,20 @@ struct _Py_slice_state {
 #endif
 };
 
+struct _Py_context_state {
+#if PyContext_MAXFREELIST > 0
```

Comment:
In the other structs you use `#ifdef WITH_FREELISTS`

Reply:
Thanks I missed it.